src/test/java/lotto/LastWeekWinnerNumberTest.java

```diff
@@ -60,7 +60,7 @@ void matchCountTest() {
 	}
 
 	@Test
-	@DisplayName("보너스 숫자 포함 여 테스트")
+	@DisplayName("보너스 숫자 포함 여부 테스트")
 	void matchBonusNumberTest() {
 		LastWeekWinnerNumber lastWeekWinnerNumber = new LastWeekWinnerNumber("1,2,3,4,5,45", new LottoNumber(33));
```

Comment:
윗 피드백과 같이 다시 테스트를 해보시면 좋을 것 같아요!
```suggestion
	@DisplayName("보너스 숫자 포함 여부 테스트")
	void matchBonusNumberTest() {
		LastWeekWinnerNumber lastWeekWinnerNumber = new LastWeekWinnerNumber("1,2,3,4,5,45", new LottoNumber(1));
```
![image](https://user-images.githubusercontent.com/20432323/113101512-c6bc0700-9237-11eb-9981-d9daab0da3e3.png)